src/amo/components/SearchResult/index.js

```diff
@@ -109,7 +133,12 @@ export class SearchResultBase extends React.Component<InternalProps> {
       <div className="SearchResult-result">
         <div className={iconWrapperClassnames}>
           {imageURL ? (
-            <img className="SearchResult-icon" src={imageURL} alt="" />
+            <img
+              className="SearchResult-icon"
+              src={imageURL}
+              alt={addon ? `${addon.name}` : ''}
```

Comment:
This is a change from before. Is this a new requirement from #5174?

Reply:
well not a requirement but on another PR this came up...that maybe it was better to have some alt rather than have it be empty. I can try to dig that comment up if that helps..